vault/request_handling.go

```diff
@@ -1130,6 +1175,11 @@ func (c *Core) handleRequest(ctx context.Context, req *logical.Request) (retResp
 					retErr = multierror.Append(retErr, ErrInternalError)
 					return nil, auth, retErr
 				}
+				defer func() {
```

Comment:
I have a suspicion that this can actually be removed as we changed the code to use ExternalID as opposed to the request.ClientToken field. Will attempt to remove it!